### DISCUSSION TITLE
【bug】在编辑器中上传了sprite后，前一天20点左右上传，第二天9点30来看，发现资源文件无法展示，问题已出现多次

### DISCUSSION THREAD

**Discussion by SQYun** [galacean/discussion#99] [Editor]
在编辑器中上传了sprite后，前一天20点左右上传，第二天9点30来看，发现资源文件无法展示，问题已出现多次
复现链接：https://galacean.antgroup.com/editor#/project/19500012

**Comment by gz65555** [comment]
已经修复
